docs/BedWarsProxy/features/placeholders.md

```diff
@@ -21,42 +22,12 @@ Placeholders that can be used on supported plugins.
-| `%bw2023_arena_status%`                   | Returns the status of the arena in the default language.                                                                                |
-| `%bw2023_player_kills%`                   | Returns the number of kills the player has done in that particular game.                                                                |
-| `%bw2023_player_kills_final%`             | Returns the number of final kills the player has done in that particular game.                                                          |
-| `%bw2023_player_kills_total%`             | Returns the total number of kills (kills + final kills) the player has done in that particular game.                                    |
-| `%bw2023_player_deaths%`                  | Returns the number of times the player has died in that particular game.                                                                |
-| `%bw2023_player_deaths_final%`            | Returns the number of times the player has died the final time in that particular game. (Supposed to be always less than or equal to 1) |
-| `%bw2023_player_deaths_total%`            | Returns the total number of times the player has died (deaths + final deaths) in that particular game.                                  |
-| `%bw2023_player_beds%`                    | Returns the number of beds the player has broken in that particular game.                                                               |
-| `%bw2023_players%`                        | Returns the players in the arena, which the player is in, in the form of a String with names separated by ", ".                         |
-| `%bw2023_players_amount%`                 | Returns the number of players in the arena the player is in.                                                                            |
-| `%bw2023_party_has%`                      | Returns "true" if the player is in a party and "false" if the player is not.                                                            |
-| `%bw2023_party_members%`                  | Returns the members of the party, which the player is in, in the form of a String with names separated by ", ".                         |
-| `%bw2023_party_members_amount%`           | Returns the number of members of the party the player is in.                                                                            |
-| `%bw2023_party_in_yours_%`                | Returns "true" if the target specified is in the player's party and "false" if not.                                                     |
-| `%bw2023_party_in_his_%`                  | Returns "true" if the player is in the target's party and "false" if not.                                                               |
-| `%bw2023_party_is_owner%`                 | Returns "true" if the player is the owner of his party and "false" if not.                                                              |
```

Comment:
You didnt like these?

Reply:
they don't even exist for me to even like them in the first place